recipes/libcublas/meta.yaml

```diff
@@ -0,0 +1,171 @@
+{% set name = "libcublas" %}
+{% set version = "12.0.1.189" %}
+{% set cuda_version = "12.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set target_name = "x86_64-linux" %}  # [linux64]
+{% set target_name = "ppc64le-linux" %}  # [ppc64le]
+{% set target_name = "sbsa-linux" %}  # [aarch64]
+{% set target_name = "x64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/{{ name }}/{{ platform }}/{{ name }}-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 38272cc9184b4a57a6250fc2cd914ffc827f9f8cd5a3e1356bc3c31f843caf8f  # [linux64]
+  sha256: 2d46e735f8e76e2eb7df1b4f41d56aa8d5d2683f99988bac82d6ae33fb6d344b  # [ppc64le]
+  sha256: 96b68f6821684f39f338862639ce74a74c75830bf44dcd17ef372a1cb8c2304d  # [aarch64]
+  sha256: 88f29bec81880844da2eee13f1cdc51e1ccd8b4c60af6df633577810b30f80d0  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+outputs:
+  - name: libcublas
+    files:
+      - lib/lib*blas*.so.*                            # [linux]
+      - targets/{{ target_name }}/lib/lib*blas*.so.*  # [linux]
+      - Library\bin\cublas*.dll                       # [win]
+      - Library\bin\nvblas*.dll                       # [win]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - sysroot_{{ target_platform }} 2.17    # [linux]
+      host:
+         - cuda-version {{ cuda_version }}
+      run:
+         - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+    test:
+      commands:
+        - test -L $PREFIX/lib/libcublas.so.{{ version }}                                          # [linux]
+        - test -L $PREFIX/lib/libcublas.so.{{ version.split(".")[0] }}                            # [linux]
```

Comment:
I've seen code that uses `dlopen` / `dlsym` as well as `ctypes` code that expects to use just the `lib*.so` as opposed to the versioned shared library at runtime. Not a blocker, but have we considered having the `libcublas.so` symlink in this package as opposed to the `dev` package? I know that the `SONAME` points to `libcublas.so.12`, but there's definitely code out there that will end up requiring `libcublas-dev` at runtime because of this.

Definitely not a blocker for getting this in though!

Reply:
We've been reworking RAPIDS code to use the proper SONAME form, like `libcublas.so.12`, anywhere that we've seen that. I could see an argument that it's a bug to rely on `dlopen`ing the unversioned form? That said, I don't have a reference for how we've packaged this in other CUDA Toolkit deployments to know if this approach is "normal" or not. I agree that we shouldn't treat this as a blocker, and I'd be happy to discuss further if users run into specific issues.

Reply:
See also:
- https://github.com/rapidsai/cudf/issues/12708
- https://github.com/rapidsai/cudf/issues/12708#issuecomment-1433366212
- https://github.com/rapidsai/cucim/issues/504

Reply:
Yeah this should be easy to add if we decide. Let's turn this into an issue on the feedstock, which will make it easier to track as well

Reply:
Raised an issue ( https://github.com/conda-forge/libcublas-feedstock/issues/1 )